src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public interface Draw {
+    public abstract void draw();
```

Comment:
Remove the redundant 'public' and 'abstract' keywords, as interface methods are implicitly public and abstract. Simply use 'void draw();' 🧹